src/persist-client/src/internal/metrics.rs

```diff
@@ -1385,12 +1406,17 @@ impl MetricsBlob {
 #[async_trait]
 impl Blob for MetricsBlob {
     async fn get(&self, key: &str) -> Result<Option<Vec<u8>>, ExternalError> {
+        let start = Instant::now();
```

Comment:
we've already got a timer inside the `run_op` call below, lets share it by passing `get_latency` in as an `Option<&Histogram>`

Reply:
As a field on `ExternalOpMetrics` or a function arg? (The former seems more consistent, if that doesn't sound too invasive?)

Reply:
Great, done!

Updated the example `/metrics` output in the description, which confirms that we're only reporting the histogram for the three opts we've opted-in. (And shows the corrected labels, etc.)